docs/test_lab.md

```diff
@@ -62,6 +62,8 @@ by the `sec_port` config (see below).
 host. 3x eXtra is recommened for a full test lab setup because it allows for running
 [core FAUCET switch tests](https://faucet.readthedocs.io/en/latest/testing.html#hardware-switch-testing-with-docker).
 At least 1 eXtra is useful for diagnosing any switch configuration problems.
+4. _Faux_ device, which can be used to run a _faux_ device under test either stand-alone or alongside other physical
```

Comment:
now that I look at this it's the same thing as the eXtra devices section above -- it's the same thing. So maybe just merge them with the additional detail.

Reply:
Formatting is weird -- see https://github.com/grafnu/daq/blob/57727816c6e2f919a045bfc72e7f656fbfc2e918/docs/test_lab.md for the rendered output. I don't think you want the <br> in there... ?